File: Day-141/ARRAY_IMPL1.cpp
vector<int> performOps(vector<int> A) {
    vector<int> B(2 * A.size(), 0);
    for (int i = 0; i < A.size(); i++) {
        B[i] = A[i];
        B[i + A.size()] = A[(A.size() - i) % A.size()];
    }
    return B;
}

File: Day-141/ARRAY_IMPL1.js
function performOps(A){
    B = new Array(2 * A.length)

    for (var i = 0; i < A.length; i++) {
        B[i] = A[i];
        B[i + A.length] = A[(A.length - i) % A.length];
    }
    return B;
}
